models/articles.models.js

```diff
@@ -13,4 +13,21 @@ exports.fetchArticleById = (id) =>{
         }
         return result.rows[0]
     })
+}
+exports.fetchArticles = (sort_by='created_at',order='desc') => {
+    let query = `SELECT articles.article_id, articles.title, articles.topic, articles.author, articles.votes, articles.created_at, COUNT(comments.article_id)::INT AS comment_count FROM articles 
+    LEFT JOIN comments ON comments.article_id = articles.article_id
+    GROUP BY articles.article_id`
+    
+    const validSortBy = ['created_at']
+    if(!validSortBy.includes(sort_by) && sort_by){
+        return Promise.reject({status: 400, msg: 'Bad Request'})
+    }
+    
+    query += ` ORDER BY articles.${sort_by} ${order.toUpperCase()};`
```

Comment:
I would get rid of `toUpperCase() `here and just make the order `'DESC'` in the parameter on line 17 👍